src/utils/configLoad.ts

```diff
@@ -59,10 +54,19 @@ const maybeInjectAnalyticsLibrary = (config: Config) => {
 };
 
 const calculateAccessModel = (config: Config): AccessModel => {
-  const { id, monthlyOffer, yearlyOffer } = config?.integrations?.cleeng || {};
+  if (config?.integrations?.cleeng) {
```

Comment:
We should check to make sure that both integrations aren't set and throw an error if they are. Otherwise trying to run with inplayer might get the wrong access model set from left over cleeng data in a config or something.